internal/olm/operator/internal/configmap.go

```diff
@@ -0,0 +1,149 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package olm
+
+import (
+	"context"
+	"crypto/md5"
+	"encoding/base32"
+	"fmt"
+	"strings"
+
+	"github.com/operator-framework/operator-sdk/internal/util/k8sutil"
+
+	"github.com/ghodss/yaml"
+	"github.com/operator-framework/operator-registry/pkg/registry"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+const (
+	// The directory containing all manifests for an operator, with the
+	// package manifest being top-level.
+	containerManifestsDir = "/registry/manifests"
+)
+
+// IsManifestDataStale checks if manifest data stored in the registry is stale
+// by comparing it to manifest data currently managed by m.
+func (m *RegistryResources) IsManifestDataStale(ctx context.Context, namespace string) (bool, error) {
```

Comment:
It has not been used outside from here. So, should it really be exported( in the upper case)?

Reply:
It will be. See #1912

Reply:
It's also in an `internal` subtree, so it isn't part of the public API either way.